common-app-lib/src/cmr/common_app/static.clj

```diff
@@ -218,6 +239,10 @@
               :headers {"Content-Type" "text/html; charset=utf-8"}
               :body (-> resource
                         slurp
+                        (string/replace "%GENERIC-TABLE-OF-CONTENTS-SEARCH-DOCS%" (gconfig/all-generic-table-of-contents gconfig/search-table-of-contents-template))
+                        (string/replace "%GENERIC-TABLE-OF-CONTENTS-INGEST-DOCS%" (gconfig/all-generic-table-of-contents gconfig/ingest-table-of-contents-template))
+                        (string/replace "%GENERIC-SEARCH-DOCS%" (read-generic-markdown "search"))
+                        (string/replace "%GENERIC-INGEST-DOCS%" (read-generic-markdown "ingest"))
```

Comment:
I would like to know if anyone else dislikes these names, specifically calling out Search and Ingest instead of just Generic-Docs or Generic-Table-of-Contents-Docs. I don't think the code will ever be called to support both cases at once, so both documents can safely use the same names. Anyone else agree?